src/application/services/useTools.ts

```diff
@@ -1,48 +1,22 @@
 import { type Ref, ref, watch } from 'vue';
 import { useAppState } from './useAppState';
+import { toolsService } from '@/domain';
 import type EditorTool from '@/domain/entities/EditorTool';
-import { loadScript } from '@/infrastructure/utils/load-script';
-import type { EditorConfig } from '@editorjs/editorjs';
-
-/**
- * Downloaded tools data structure
- */
-type DownloadedTools = EditorConfig['tools'];
+import type { EditorConfigTools } from '@/domain/entities/EditorTool';
 
 /**
  * Service for load editor tools
  *
- * @param noteTools - note tools
+ * @param tools - note tools
  */
-export function useTools(noteTools: Ref<EditorTool[]>): {
-  tools: Ref<DownloadedTools | undefined>;
+export function useTools(tools: Ref<EditorTool[]>): {
```

Comment:
the interface looks unclear:

```ts
const { tools } = useTools(note.tools);
```

Reply:
is this service responsible only for note tools?

Reply:
I don't understand. Now it's hooks only for note, but as we said it can be used for every tools

Reply:
you need to improve naming or interface to make it more straightforward. 

```
const { tools } = useTools(note.tools);
```

You're passing `note.tools` to some `useTools()` function. And get the `tools` returned. It's unclear what is happening here.

Reply:
```ts
const { noteAndUserTools } = useTools(note.tools);
```